packages/epo-widget-lib/src/widgets/CameraFilter/CameraFilter.tsx

```diff
@@ -0,0 +1,123 @@
+import { FunctionComponent, useState } from "react";
+import { SelectListbox, tokens } from "@rubin-epo/epo-react-lib";
+import { useTranslation } from "react-i18next";
+import useResizeObserver from "use-resize-observer";
+import { filters, Filter, rangeConfig } from "./data";
+import * as Styled from "./styles";
+import CondensedFilterRanges from "./CondensedFilterRanges";
+import SpectrumDisplay from "./SpectrumDisplay";
+
+const CameraFilter: FunctionComponent = () => {
+  const { t } = useTranslation();
+  const { ref, width = 1 } = useResizeObserver<HTMLDivElement>();
+  const [activeFilterBand, setActiveFilterBand] = useState<string | null>(null);
+  const { BREAK_TABLET } = tokens;
+  const isCondensed = width < parseInt(BREAK_TABLET);
+
+  const {
+    [isCondensed ? "condensed" : "default"]: { min, max, range: spectrumRange },
+  } = rangeConfig;
+
+  const filterMin = filters[0].range[0];
+  const filterMax = filters[filters.length - 1].range[1];
+  const filterRange = filterMax - filterMin;
+
+  const filtersOnly = filters.filter(({ band }) => band) as Required<Filter>[];
+
+  const noneOption = {
+    value: "none",
+    label: t("camera_filter.labels.option", { context: "none" }),
+  };
+  const options = [noneOption].concat(
+    filtersOnly.map(({ band }) => ({
+      value: band,
+      label: t("camera_filter.labels.option", { band }),
+    }))
+  );
+
+  const activeFilter = filters.find(({ band }) => band === activeFilterBand);
+
+  return (
+    <Styled.FilterContainer ref={ref}>
+      <h1>{t("camera_filter.title")}</h1>
+      {isCondensed && (
+        <CondensedFilterRanges
+          filters={filtersOnly}
+          {...{ min, spectrumRange }}
+        />
+      )}
+      <Styled.FilterTable
+        aria-hidden={isCondensed}
```

Comment:
table is hidden from accessibility tree in screen readers since it is reproduced by the labels + some ARIA labelled-by magic